src/components/common/navigation/NavigationRoutes.tsx

```diff
@@ -404,7 +404,7 @@ export default function NavigationRoutes() {
                                                 />
                                             </Route>,
                                         ]}
-                                        {isSuperAdmin && !window._env_.K8S_CLIENT && (
+                                        {!window._env_.K8S_CLIENT && (
```

Comment:
Question: Are we opening the route for non super admins as well?

Reply:
yes